src/index.html

```diff
@@ -9,6 +9,40 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="logo" href="#home">
+        <img src="./images/logo.png" alt="logo">
+      </a>
+      <nav>
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="is-active" href="#">Apple</a>
```

Comment:
Each link should be an anchor link with corresponding address, not just `#`
```suggestion
            <a class="is-active" href="#apple">Apple</a>
```